docs/source/site.md

```diff
@@ -0,0 +1,72 @@
+---
+html_meta:
+  "description": "Site endpoint for Plone REST API"
+  "property=og:title": "Site endpoint for Plone REST API"
+  "property=og:description": "Site endpoint for Plone REST API"
+  "keywords": "Plone, plone.restapi, REST, API, site, navigation root"
+---
+
+# Site
+
+The `@site` endpoint provides site-wide information, such as the site title, logo, and other information, which is useful to offer generic information about the Plone site.
+
+Send a `GET` request to the `@site` endpoint:
+
+```{eval-rst}
+..  http:example:: curl httpie python-requests
+    :request: ../../src/plone/restapi/tests/http-examples/site_get.req
+```
+
+The response will contain the site information:
+
+```{literalinclude} ../../src/plone/restapi/tests/http-examples/site_get.resp
+:language: http
+```
+
+## Navigation root
+
+Plone has an idea called `Navigation Root` which provides a way to root catalog queries, searches, breadcrumbs, etc. in a given section of the site. This feature is useful when working with subsites or multilingual sites, because allows the site manager to somehow restrict where the searches or navigation queries start on the site.
```

Comment:
This content appears to have been duplicated from `navroot.md`. I'm not sure why that is necessary. Can you elaborate?

If it is necessary, then please apply the changes and suggestions from that section to this one, so that I don't have to repeat my review.